docs/releases.md

```diff
@@ -30,6 +30,44 @@
 
 ## Version 6
 
+### v6.6.0
+
+Released on May 1, 2025
+
+#### Legacy is gone and improved image security
+
+To put it simply, we are getting rid of the legacy v4 code.
+This change is significant for two reasons. First of all, it means that the API v1 is now completely gone.
+But more importantly, this is a big change for us, as we have been using the v4 codebase as a safety net for a long time.
+If something breaks, there will no longer be an easy flag that disables the complete vue-js front-end and fallbacks to the v4 codebase.
+
+This fallback process has been extremely useful to us when we deployed the version 5, and in early version 6. However we are now satisfied with the current state of the front-end and we no longer think that the fallback to v4 is needed.
+
+One may wonder why we decided to do this change, and to that there are two big reasons:
+
+- We no longer will need to maintain a legacy compatibility. Cleaning old code bases is part of the life cycle of a project, and we are now at the point where we can do this. The second reason is that we are planning to refactor the way images are attached to albums.
+- We decided to go in the direction of a many-to-many relationship. In layman terms, this means that a photo can be in multiple albums at the same time. This behaviour is not compatible with the API v1 definitions. This relationship change is going to have significant impact on the way photos access rights are handled, but we are confident this will also enable us to provide more interesting features in the future.
```

Comment:
```suggestion
- We no longer will need to maintain a legacy compatibility. Cleaning old code bases is part of the life cycle of a project, and we are now at the point where we can do this.
- We are planning to refactor the way images are attached to albums. We decided to go in the direction of a many-to-many relationship. In layman terms, this means that a photo can be in multiple albums at the same time. This behaviour is not compatible with the API v1 definitions. This relationship change is going to have significant impact on the way photos access rights are handled, but we are confident this will also enable us to provide more interesting features in the future.
```